packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -55,11 +55,17 @@ export const InputField = ({ name, optional = false, ...props }) => (
     </Field>
 );
 
-const validatePersonalDetails = (values, residence_list, account_opening_reason) => {
+const validatePersonalDetails = (values, residence_list, account_opening_reason, landing_company) => {
```

Comment:
since this function takes quite a few params now we could:

```javascript
({ values, residence_list, account_opening_reason, landing_company })
```